FWCore/MessageService/src/ThreadSafeLogMessageLoggerScribe.cc

```diff
@@ -885,6 +885,15 @@ namespace edm {
         if (m_statisticsResets[i])
           m_statisticsDestControls[i]->wipe();
       }
+      auto dropped = m_tooManyWaitingMessagesCount.load();
+      if (m_statisticsDestControls.empty() and dropped != 0) {
+        if (m_stream_ps.find("cerr") != m_stream_ps.end()) {
+          std::cerr << "MessageLogger: dropped waiting message count " << dropped << "\n";
+        }
+        if (m_stream_ps.find("cout") != m_stream_ps.end()) {
+          std::cout << "MessageLogger: dropped waiting message count " << dropped << "\n";
+        }
```

Comment:
What if `m_stream_ps` contains both `cerr` and `cout`? Would it still be appropriate to print the message in both?

Reply:
If that would be the case (and I don't think I've found a case where that is actually done) maybe they are redirecting output to two different files than both messages could still be useful. But I'm not wedded to the idea.

Reply:
@makortel Do you want to have this changed to only output to cerr if both cerr and cout are active or keep it as it is?

Reply:
Let's leave it as it is.